src/swap/core/getSwapMessage.ts

```diff
@@ -39,14 +52,28 @@ export function getSwapMessage({
   if (!error) {
     return '';
   }
-  // handle invalid params error
+  // handle specific error codes
+  if (
+    Object.values(error)?.find(
```

Comment:
All `Object.values(error)?.find(` could be an utility function. And make this code cleaner.

Reply:
added `getSwapError`